### DISCUSSION TITLE
Set app as home screen

### DISCUSSION THREAD

**Discussion by dkmcgowan** [thecowan/wallpanel-android#323] [Q&A] [answered]
I originally chose to not do this, picked always for the normal Home app, now I wanted to try to set WallPanel as the home app to help it restart when it stops, but can't find the settting in Android anywhere.  Once you choose one and do always, how do you get it back?  I see in settings how to clear it, but not set it

**Comment by thanksmister** [comment, marked as answer]
The Android system sets the home screen and its really difficult to undo it.  I put a setting in WallPanel to reset the home screen on the device, use this, then close and relaunch WallPanel, it should ask again what you want to use for the home screen.

**Reply by jmdv-es** [reply]
thanks, i was really struggling with this
